app/controllers/content.js

```diff
@@ -127,6 +128,18 @@ export default Ember.Controller.extend(Analytics, {
         return text;
     }),
 
+    useShortenedDescription: Ember.computed('node.description', function() {
+        return this.get('node.description').length > 350;
+    }),
+
+    shortenedDescription: Ember.computed('node.description', function() {
```

Comment:
It seems that `useShortenedDescription` could be removed, in favor of a single property (`description`) that just chose what text to display based on length. This would let you consolidate some redundant markup in `content.hbs` as well.